testSettings.py

```diff
@@ -59,7 +60,7 @@ def _is_same_node_test(self, test_id: int) -> bool:
 
     def _server_test_to_persistent(self, test_id: int) -> bool:
         #TODO: add logic to determine when this is required
-        return False
+        return True
```

Comment:
Let's try to run the tests without persistent at first. I think it is the harder option.